Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ImageEx/ImageExPage.xaml.cs

```diff
@@ -67,6 +69,17 @@ private async void Load()
                 AddImage(false, false, true);
             });
 
+            SampleController.Current.RegisterNewCommand("Lazy loading sample", (sender, args) =>
+            {
+                var imageExLazyLoadingControl = new ImageExLazyLoadingControl();
+                imageExLazyLoadingControl.CloseButtonClick += (_, __) =>
+                {
+                    lazyLoadingControlHost.Child = null;
```

Comment:
This is not guaranteed to be set if the user removes it from XAML, need to do null check for `lazyLoadingControlHost` here and below.